.github/workflows/pytest.yaml

```diff
@@ -50,7 +50,7 @@ jobs:
       run: mypy broadbean
     - name: Run tests
       run: |
-        pytest --cov=broadbean --cov-report xml --hypothesis-profile ci
```

Comment:
`-cov-append` is not needed here. That is only used in the qcodes tests because we run them as 2 separate sets and want to merge the second into the first